src/nodes.coffee

```diff
@@ -1501,11 +1510,15 @@ exports.Access = class Access extends Base
     else
       [@makeCode('['), name..., @makeCode(']')]
 
+  shouldCache: NO
+
   ast: ->
+    # Babel doesn’t have an AST node for `Access`, but rather just includes
```

Comment:
This comment isn't particularly clarifying imo. If even worth having here, it could be a lot more specific as I think an `Access` will always correspond to the `property` of a `MemberExpression`